src/config/index.js

```diff
@@ -9,13 +9,18 @@ import Shibuya from './shibuya';
 import Rocstar from './rocstar';
 import Shiden from './shiden';
 
+import MoonbaseDev from './moonbase-dev';
+import MoonbaseAlpha from './moonbase-alpha';
+
 import TuringDev from './turing-dev';
 import TuringStaging from './turing-staging';
+import TuringMoonbaseAlpha from './turing-moonbase-alpha';
 import Turing from './turing';
 
 export {
     Rococo, RococoDev,
     MangataDev, MangataRococo, Mangata,
-    TuringDev, Turing, TuringStaging,
+    TuringDev, Turing, TuringStaging, TuringMoonbaseAlpha,
```

Comment:
Let’s call it `TuringMoonbase`. The three chains in local network should be named,

MoonbaseAlphaRelay
MoonbaseAlpha
TuringMoonbase

Reply:
Solved!